src/constants/text.js

```diff
@@ -107,5 +107,9 @@ export default {
   zeroSelected: "0 selected",
   showEvents,
   clearAll: "Clear all",
-  filterEvents: "Filter Events"
+  filterEvents: "Filter Events",
+  noSavedEventsTitle: "Where is the love?",
+  noSavedEventsPart1: "No favourites yet? Tap the",
+  noSavedEventsPart2: "on any events you like to save them here.",
+  noSavedEventsButton: "Find Events"
```

Comment:
To make this more discoverable, I suggest starting these with the screen name (`savedEvents...`) for example `savedEventsEmptyTitle`.